### PR TITLE
binding data, update UI for username and bio

### PR DESCRIPTION
**Pull request by SYBERTOT**

### CODE REVIEW

File: app/src/main/java/com/example/murom/ProfileFragment.java [hunk 1]
```diff
@@ -106,6 +109,19 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
...
+                String userName = documentSnapshot.getString("username");
+                String userBio = documentSnapshot.getString("bio");
+                username.setText(userName);
+                bio.setText(userBio);
+            } else {
+                Log.d("-->", "User document does not exist");
+            }
+        }).addOnFailureListener(e -> {
+            Log.d("-->", "Failed to fetch user data: " + e.getMessage());
+        });
```
Comment (devtronghieu) [inline app/src/main/java/com/example/murom/ProfileFragment.java:124]: use `ProfileState.getInstance().profile`

File: app/src/main/java/com/example/murom/EditProfileFragment.java [hunk 2]
```diff
@@ -119,4 +130,29 @@ private void handleSignOut(View view) {
         startActivity(i);
     }
 
+    private void saveChanges(String newUsername, String newDescription) {
```
Comment (ltminhthu) [inline app/src/main/java/com/example/murom/EditProfileFragment.java:133]: @SYBERTOT  saveChanges nay nen truyen vo mot cai ham luu len database (ham do viet cho database chu hong nen viet ngay day nhe)

File: app/src/main/java/com/example/murom/ProfileFragment.java [hunk 3]
```diff
@@ -106,6 +110,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
                     Log.d("-->", "failed to get avatar: " + e);
                 });
 
+        String userName = ProfileState.getInstance().profile.username;
```
Comment (ltminhthu) [inline app/src/main/java/com/example/murom/ProfileFragment.java:113]: @SYBERTOT  cho nay goi cai bien `profile` roi assign no thanh `ProfileState.getInstance().profile` xong ui xai lai bien `profile`  thui nhe

File: app/src/main/res/layout/fragment_profile.xml [hunk 4]
```diff
@@ -168,13 +168,17 @@
 
     <androidx.constraintlayout.widget.ConstraintLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content">
+        android:layout_height="wrap_content"
+        android:layout_marginTop="10dp">
```
Comment (ltminhthu) [inline app/src/main/res/layout/fragment_profile.xml:172]: should remove marginTop here

File: app/src/main/java/com/example/murom/EditProfileFragment.java [hunk 5]
```diff
@@ -24,12 +24,21 @@
...
+import com.google.firebase.firestore.SetOptions;
 import com.google.firebase.storage.StorageReference;
 
+import java.util.HashMap;
+
 
 public class EditProfileFragment extends Fragment {
 
+    ProfileState profileState = ProfileState.getInstance();
+    String currentUserUid = Auth.getUser().getUid();
```
Comment (devtronghieu) [inline app/src/main/java/com/example/murom/EditProfileFragment.java:41]: use `String currentUserUid = profileState.profile.id`

**Review by devtronghieu — APPROVED** [review]

**Pull request merged**